packages/block-editor/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 -   `URLInput`: the `renderSuggestions` callback prop now receives `currentInputValue` as a new parameter ([45806](https://github.com/WordPress/gutenberg/pull/45806)).
```

Comment:
I see that `URLInput` is included in the changelog already. Should I have added my PR below, or is it fine the way I did it?

Reply:
This way is fine 👍